lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -352,7 +352,7 @@
       - Connect the container to a network. Choices are "bridge", "host", "none" or "container:<name|id>"
   userns_mode:
      description:
-       - User namespace to use
+       - Set the usernamespace mode for the container. Only valid value is "host".
```

Comment:
```suggestion
       - Set the user namespace mode for the container. Currently, the only valid value is C(host).
```

Reply:
I think this formulation is a bit better. In particularly because docker might add other values for this option in the future :)

Reply:
sure, fine with me:)

Reply:
Can you update the PR accordingly? I guess we can merge it then.

Reply:
@TimoKramer you should see a button for accepting @felixfontein's suggestion on your PR - if you accept the suggestion, I can merge this PR. Thanks!